src/internet_identity/src/assets.rs

```diff
@@ -55,21 +58,19 @@ lazy_static! {
         format!("sha256-{hash}")
     };
 
-    // Various HTML pages, after the canister ID, the script tag and the CSP have been injected
-
-    static ref INDEX_HTML_STR: String = {
-        fixup_html(include_str!("../../../dist/index.html"))
-    };
-
-    static ref ABOUT_HTML_STR: String = {
-        fixup_html(include_str!("../../../dist/about.html"))
-    };
+    /// Map of path aliases
+    /// This creates a copy of the asset at the mapped path with the same content as the asset at the
+    /// original path.
+    static ref PATH_ALIASES: HashMap<String, String> = vec![
+        ("/index.html".to_string(), "/".to_string()),
```

Comment:
idea: how about mapping `**/index.html` to `**/` and `**/<foo>.html` to `**/<foo>` in general? I think that's what most HTTP servers do (module some details like how they handle `/foo` vs `/foo/`).

Reply:
Good idea 👍 
Done.